src/racer.c

```diff
@@ -875,21 +866,19 @@ void func_8004447C(Object *aiRacerObj, Object_Racer *aiRacer, s32 updateRate) {
                 sp4E = aiRacer->elevation;
                 tempRacerObj =
                     ainode_get(aiRacer->unk1CE); // I'm assuming this is a Ai Node (Take with a grain of salt!)
-                raceType = tempRacerObj->segment.level_entry->aiNode.elevation;
-                someBool = FALSE;
-                if (sp6C->aiNode.elevation < raceType) {
-                    if ((raceType < sp4E) || (sp4E < sp6C->aiNode.elevation)) {
-                        someBool = TRUE;
+                newvar = tempRacerObj->segment.level_entry;
+                sp46 = FALSE;
+                if (sp6C->aiNode.elevation < newvar->aiNode.elevation) {
+                    if (1) {} // FAKE
+                    if (newvar->aiNode.elevation < sp4E || sp4E < sp6C->aiNode.elevation) {
+                        sp46 = TRUE;
                     }
                 } else {
```

Comment:
Can this be made an `else if (sp6C->aiNode.elevation < sp4E || sp4E < newvar->aiNode.elevation)` now?